src/components/TaskList/TaskList.jsx

```diff
@@ -219,7 +243,9 @@ class TaskList extends React.Component {
           </div>
         </nav>
         <div className="task-list-container">
-          {this.taskListOrMessage()}
+          <div className={this.state.showHidden && 'viewing-hidden'}>
```

Comment:
Does this result in the class being nothing if showHidden is false, and the class being 'viewing-hidden' when showHidden is true?